{{ cookiecutter.__repository }}/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,33 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-dockerfile
+{
+	"name": "Existing Dockerfile",
+	"build": {
+		// Sets the run context to one level up instead of the .devcontainer folder.
+		"context": "..",
+		// Update the 'dockerFile' property if you aren't using the standard 'Dockerfile' filename.
+		"dockerfile": "../Dockerfile"
+	},
+	"containerEnv": {
+		"SHELL": "/bin/bash"
+	},
+	"postCreateCommand": "pip install -r tests/requirements.txt",
+	"customizations": {
+		"vscode": {
+			"extensions": [
```

Comment:
Should we extract those and add them to a newly created .vscode/extensions.json?
Is there a way for get the devcontainer to force install from there, so that they are in sync with those that would be applied on non-devcontainer setups?